setup.py

```diff
@@ -2,16 +2,16 @@
 
 setup(
     name="imhotep",
-    version="3.0.0",
+    version="3.0.1",
     packages=find_packages(),
-    url="https://github.com/justinabrahms/imhotep",
+    url="https://github.com/eBay/imhotep",
```

Comment:
Please revert this. The package is mine, even if eBay has a fork.

Reply:
Ah. My apologies. I misunderstood your intention.